OracleWebCenterPortal/dockerfiles/README.md

```diff
@@ -59,7 +61,7 @@ $ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.
 ```
 
 
-The Oracle Database image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
+The Oracle Database image can be pulled from the  [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
```

Comment:
Please replace "in [GitHub Location]" with "from the [Oracle Database](https://github.com/oracle/docker-images/tree/master/OracleDatabase) section of this repo."